modules/programs/fish.nix

```
@@ -430,6 +430,14 @@ in {
               set fish_complete_path $fish_complete_path[1] $plugin_dir/completions $fish_complete_path[2..-1]
             end
 
+            if test -d $plugin_dir/themes
+              # https://github.com/fish-shell/fish-shell/issues/9456
+              mkdir -p $__fish_config_dir/themes
+              for f in $plugin_dir/themes/*.theme
+                ln -s $f $__fish_config_dir/themes/(basename $f)
```

If the link exists from a previous run, wouldn't this cause an error message? Perhaps need to force the link creation?

```suggestion
                ln -sf $f $__fish_config_dir/themes/(basename $f)
```

---

indeed, thanks for the review!